packages/frontend/src/pages/welcome.entrance.a.vue

```diff
@@ -59,6 +59,7 @@ function getInstanceIcon(instance: Misskey.entities.FederationInstance): string
 misskeyApiGet('federation/instances', {
 	sort: '+pubSub',
 	limit: 20,
+	blocked: 'false'
```

Comment:
```suggestion
	blocked: false,
```

- corrected code style for lint
- corrected types

Reply:
Could this result in the value passed in being `1`?

Reply:
I think you are right, sorry!

(Be sure to add trailing comma, though)

Reply:
done